src/domains/ecommerce-common/02_security.tf

```diff
@@ -741,6 +741,28 @@ resource "azurerm_key_vault_secret" "ecommerce_payment_requests_secondary_api_ke
   key_vault_id = module.key_vault.id
 }
 
+resource "random_password" "ecommerce_notification_service_primary_api_key_pass" {
+  length  = 32
+  special = false
```

Comment:
please add keepers section here to prepare to key rotation when needed.
Take a look [here](https://github.com/pagopa/pagopa-infra/blob/202e0660cff4c312e96c06575a911fb3f3ce8de2/src/domains/ecommerce-common/02_security.tf#L852-L855)